cmd/post_templates/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/checkr/openmock"
+	"github.com/sirupsen/logrus"
+)
+
+type Config struct {
+	URL string // this is a field
+}
+
+func main() {
+	config := &Config{URL: os.Getenv("OPENMOCK_URL")}
+
+	om := &openmock.OpenMock{}
+	om.ParseEnv()
+
+	om.SetupLogrus()
+	om.SetupRepo()
+
+	err := om.Load()
+	if err != nil {
+		logrus.Fatalf("%s: %s", "failed to load yaml templates for mocks", err)
+	}
+
+	// post the loaded templates at the openmock instance
+	templatesYaml := om.ToYAML()
+	response, httpErr := http.Post(config.URL, "application/yaml", bytes.NewReader(templatesYaml))
+	if httpErr != nil {
+		fmt.Printf("Error posting templates at %s: [%s]", config.URL, httpErr)
```

Comment:
btw, you can also just use `panic("...")` or `log.Fatalf`.

Reply:
👍